docs/multi-sport-scoring-job-refactor.md

```
@@ -0,0 +1,182 @@
+    unscoredContestIds = UserPicks.Where(ScoredAt == null).Select(ContestId).Distinct()
+    For each contest in (unscored ∩ finalized):
+        Enqueue ContestScoringProcessor
+```
+
+Discover the active sport list at runtime from `PickemGroups.Select(g => g.Sport).Distinct()` so adding a new sport doesn't require touching this job. Daily cadence.
+
+### 6. `LeagueWeekScoringJob` — sport-agnostic daily backstop
+
+Rewrite to:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)